inference-engine/include/ie_iextension.h

```diff
@@ -198,9 +207,7 @@ using IExtensionPtr = std::shared_ptr<IExtension>;
  * @brief Creates the default instance of the extension
  *
  * @param ext Extension interface
- * @param resp Response description
- * @return Status code
  */
-INFERENCE_EXTENSION_API(StatusCode) CreateExtension(IExtension*& ext, ResponseDesc* resp) noexcept;
+INFERENCE_EXTENSION_API(void) CreateExtension(IExtensionPtr& ext);
```

Comment:
how will it work for existing extensions? I suppose it will break compatibility..

Reply:
Seems you right